### DISCUSSION TITLE
Request for more info in Offer creation dialog

### DISCUSSION THREAD

**Discussion by danieljperry** [Chia-Network/chia-blockchain-gui#1083] [Ideas]
New users likely won't understand how offer files work. Currently the Offer Builder screen has a `CREATE OFFER` button. A new user might think that clicking this button will either exchange the assets automatically, or that it will at least enable such an exchange. In reality, it only results in a new dialog popping up, and even that dialog just creates a new file, which then needs to be sent out for any exchange to take place.

A few suggestions, some of which might negate the need for others:
* In the `Offer Management` screen, change the name of the `CREATE AN OFFER` button to `BUILD AN OFFER`
* In the `Offer Builder` screen, change the name of the `CREATE OFFER` button to `REVIEW OFFER`
* Change the name of the next `CREATE OFFER` dialog that pops up to `REVIEW OFFER`
* Add a summary of what is being offered in the `REVIEW OFFER` dialog, similar to how it looks in the `Offered` and `Requested` columns in the `Offer Management` screen. The reason for this suggestion is because for offers of NFTs and multiple assets, it can be visually difficult to discern quickly everything that is being offered. So this would be a good place for a text-only summary. For example:
<html>
<body>

Offered | Requested
-- | --
5 CAT A | 1 XCH
12 CAT B | 100 SBX
NFT A | NFT C
NFT B

</body>
</html>

</body>
</html>

* Add a note to the `Review Offer` dialog explaining that this will merely create a file; it won't be possible for anyone to accept the offer until you send it somewhere, which you will be able to do in the next step
* Add to the `Share Offer` dialog a message stating that the offer was created successfully, now you must share it in order for it to have a chance of being accepted
* Somewhere in these screens/dialogs include a tooltip to get more info about offers from our website

**Comment by paninaro** [comment]
@sethjenks for visibility
